src/bin/inx-chronicle/error.rs

```diff
@@ -17,4 +17,6 @@ pub enum Error {
     #[cfg(feature = "inx")]
     #[error(transparent)]
     Inx(#[from] super::stardust_inx::InxError),
+    #[error(transparent)]
+    Shutdown(#[from] tokio::sync::broadcast::error::SendError<()>),
```

Comment:
Maybe we can just use a panic, instead of introducing a new error variant?

Reply:
Let's just assume we're going to refactor later using a different error lib.